kpi-tracker/src/main.rs

```diff
@@ -288,26 +293,29 @@ fn to_block_events(block_hash: BlockHash, block_item: BlockItemSummary) -> Vec<B
             match &atd.effects {
                 AccountTransactionEffects::ModuleDeployed { module_ref } => {
                     let details = ContractModuleDetails { block_hash };
-                    let event = BlockEvent::ContractModuleDeployment(*module_ref, details);
+                    log::debug!("CONTRACT MODULE: {}\n{:?}", module_ref, &details);
 
+                    let event = BlockEvent::ContractModuleDeployment(*module_ref, details);
                     events.push(event);
                 }
                 AccountTransactionEffects::ContractInitialized { data } => {
                     let details = ContractInstanceDetails {
                         block_hash,
                         module_ref: data.origin_ref,
                     };
-                    let event = BlockEvent::ContractInstantiation(data.address, details);
+                    log::debug!("CONTRACT INSTANCE: {}\n{:?}", data.address, &details);
 
+                    let event = BlockEvent::ContractInstantiation(data.address, details);
                     events.push(event);
                 }
                 _ => {}
             };
         }
         AccountCreation(acd) => {
             let details = account_details(block_hash, acd);
-            let block_event = BlockEvent::AccountCreation(acd.address, details);
+            log::debug!("ACCOUNT: {}\n{:?}", acd.address, &details);
```

Comment:
I guess you find this useful for debugging, but I don't think these big logs should stay there. This is way too much data. We can keep this in now and do a final pass once it's to be merged into main.

Reply:
Yes, this was mainly used to verify that I actually got the data I was expecting. I'll remove it, since it's not useful anymore.